generator/generator_tool/generator_tool.cpp

```diff
@@ -100,7 +100,8 @@ DEFINE_string(osm_file_name, "", "Input osm area file.");
 DEFINE_string(osm_file_type, "xml", "Input osm area file type [xml, o5m].");
 DEFINE_string(data_path, "", GetDataPathHelp());
 DEFINE_string(user_resource_path, "", "User defined resource path for classificator.txt and etc.");
-DEFINE_string(intermediate_data_path, "", "Path to stored nodes, ways, relations.");
+DEFINE_string(intermediate_data_path, "", "Path to stored intermediate data.");
+DEFINE_string(cache_path, "", "Path to stored caches for nodes, ways, relations.");
```

Comment:
maybe write "intermediateDir if empty" or something along these lines

Reply:
++ If 'cache_path' is empty, caches is stored to 'intermediate_data_path'